doc/source/deployment/setup-deployer.rst

```diff
@@ -46,7 +46,8 @@ The following software must be installed on your `Deployer`:
   * gcc
   * git
   * jq
-  * python3-netaddr
+  * python-jmespath
+  * python-netaddr
```

Comment:
Why this? Package has explicit python3 requirements, see https://build.opensuse.org/package/view_file/Cloud:socok8s/socok8s/socok8s.spec?expand=1

And we just merged another PR that runs explicit `python3` when importing modules: https://github.com/SUSE-Cloud/socok8s/pull/426

Reply:
Thanks, let me fix that.

Reply:
I still like to install those myself, so I think it's still worth mentioning. I don't know why it was removed or if I forgot it in the past.

Reply:
This should be mentioned but probably in a different part? like development? 

Because we are using package for installation, its the package the one that should have those requirements included and pull those, you should not need to install the package + extras for basic functionality!